Hashids/Hashids.swift

```diff
@@ -50,230 +48,204 @@ public typealias Hashids = Hashids_<UInt32>
 
 // MARK: Hashids generic class
 
-public class Hashids_<T: protocol<Equatable, UnsignedIntegerType>> : HashidsGenerator
-{
-    public typealias Char = T;
+public class Hashids_<T: protocol<Equatable, UnsignedIntegerType>>: HashidsGenerator {
+    public typealias Char = T
     
-    private var minHashLength:UInt;
+    private var minHashLength: UInt
     
-    private var alphabet:[Char];
+    private var alphabet: [Char]
     
-    private var seps:[Char];
+    private var seps: [Char]
 
-    private var salt:[Char];
+    private var salt: [Char]
     
-    private var guards:[Char];
+    private var guards: [Char]
     
-    public init(salt:String!, minHashLength:UInt = 0, alphabet:String? = nil)
-    {
-        var _alphabet = (alphabet != nil) ? alphabet! : HashidsOptions.ALPHABET;
-        var _seps = HashidsOptions.SEPARATORS;
+    public init(salt: String, minHashLength: UInt = 0, alphabet: String? = nil) {
+        let _alphabet = alphabet ?? HashidsOptions.ALPHABET
+        var _seps = HashidsOptions.SEPARATORS
         
-        self.minHashLength = minHashLength;
-        self.guards = [Char]();
-        self.salt = salt.unicodeScalars.map{ numericCast($0.value) };
-        self.seps = _seps.unicodeScalars.map{ numericCast($0.value) };
-        self.alphabet = unique( _alphabet.unicodeScalars.map{ numericCast($0.value) } );
+        self.minHashLength = minHashLength
+        self.guards = [Char]()
+        self.salt = salt.unicodeScalars.map{ numericCast($0.value) }
+        self.seps = _seps.unicodeScalars.map{ numericCast($0.value) }
+        self.alphabet = unique( _alphabet.unicodeScalars.map{ numericCast($0.value) } )
         
-        self.seps = intersection(self.alphabet, self.seps);
-        self.alphabet = difference(self.alphabet, self.seps);
-        shuffle(&self.seps, salt: self.salt);
+        self.seps = intersection(self.alphabet, self.seps)
+        self.alphabet = difference(self.alphabet, self.seps)
+        shuffle(&self.seps, salt: self.salt)
 
         
-        let sepsLength = self.seps.count;
-        let alphabetLength = self.alphabet.count;
+        let sepsLength = self.seps.count
+        let alphabetLength = self.alphabet.count
         
-        if ( (0 == sepsLength) || (Double(alphabetLength) / Double(sepsLength) > HashidsOptions.SEP_DIV) ) {
+        if 0 == sepsLength || Double(alphabetLength) / Double(sepsLength) > HashidsOptions.SEP_DIV {
```

Comment:
Parens separating these out might actually be nice here, just not the outside parens for the if